### DISCUSSION TITLE
Editable Table

### DISCUSSION THREAD

**Discussion by pmosconi** [cloudscape-design/components#597] [Q&A] [answered]
Hi,
I'd like to receive feedback about editable table: is it good to use and when from a UX perspective?
If yes, does the Table component supports it and how?
Many thanks and great work!

**Comment by pan-kot** [comment, marked as answer]
Hello Paolo,

Yes, the table with editable cells is a valid pattern.

We are working on documenting the pattern and it will include certain changes to the table component. The existing table implementation does not prevent you from rendering editable fields to cells, so I'd say we have a limited support for now.

**Reply by pmosconi** [reply]
Hi Andrei,
thanks for your reply: do you already have a tentative timeframe for the update (days, weeks, months,...)?

**Reply by just-boris** [reply]
Hello!

I want to add some clarification bits here.

1. [The standard pattern](https://cloudscape.design/patterns/resource-management/edit/) with full-page form for edit is still the recommended pattern and this will stay in the future
2. There will be inline editing support for some rare circumstances when you want to edit 1-2 fields in a list, but the preferred path does not change

To give more detailed advice, we need to know more info about your case

**Reply by pmosconi** [reply]
Hi Boris,
I need to replace an existing application where you can update inline a full row composed of over 20 fields: imagine creating or modifying ec2 or rds instances inline.
When repeating actions over several rows, this can be quite efficient.

**Reply by just-boris** [reply]
I'd recommend checking out the entire ["Resource management" section](https://cloudscape.design/patterns/resource-management/create/) on our website, there are also relevant [demo pages](https://cloudscape.design/demos/overview/).

The inline editing feature mentioned by @pan-kot does not apply for your case, because of number of editable fields.
